docs/ManualDeploymentGuideScalarDLAuditorOnAzure.md

```diff
@@ -0,0 +1,141 @@
+# Deploy Scalar DL Auditor on Azure
+
+This guide shows you how to manually deploy Scalar DL Auditor on a managed database service and a managed Kubernetes service in Azure as part of deploying Scalar DL for production.
+If you have not deployed Scalar DL Ledger, please follow [Deploy Scalar DL on Azure](ManualDeploymentGuideScalarDLOnAzure.md) guide.
+
+## What we create
+
+![image](images/auditor-arch.png)
+
+In this guide, we will create the following components for auditor.
+
+* An Azure Virtual Network associated with a Resource Group.
+* An AKS cluster with 2 node pools.
+* A managed database service.
+* A Cosmos DB Account.
+* A Bastion instance with a public IP.
+* Azure container insights.
+
+## Prerequisites
+
+* Scalar DL Ledger deployment with auditor configuration completed.
+
+## Step 1. Create an environment
+
+Scalar DL Auditor component is implemented to detect Byzantine faults,
+it manages the identical states of Ledger so you need to deploy Ledger and Auditor in separate administrative domains. For more details, please refer to  [Getting Started with Scalar DL Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md) guide.
+
+In this guide, for ease of explanation, we deploy Auditor in a different cluster on the same subscription. However, it is highly recommended to deploy it in another administrative domain in production.
+
+### Recommendations
+
+* You should deploy Ledger and Auditor in separate administrative domains.
+
+### Steps
+
+* Follow step 1 to step 3 in the [Deploy Scalar DL on Azure](./ManualDeploymentGuideScalarDLOnAzure.md) guide to create an Scalar DL Auditor environment.
+
+## Step 2. Peer the Virtual Networks
+
+This section shows how to peer the virtual networks for Scalar DL deployment.
+
+In this guide, Ledger and Auditor are deployed on the private subnet of the different virtual networks, 
+so you need to create the peering for internal communication between the Auditor, Ledger and Client SDK(application). 
+
+### Requirements
+
+* You must create a peering between the virtual networks created for Ledger and Auditor services.
+* You must create a peering between the virtual network created for Ledger and Client SDK (application).
+* You must create a peering between the virtual network created for Auditor and Client SDK (application).
+
+### Steps
+
+* Create the peering on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering) with the above requirements.
```

Comment:
If we need 3 peerings, have 3 steps here.

Reply:
Thank you for your review comments.
Document updated with the above changes 9ea12d6